### ISSUE TITLE
Illegal UTF-8 encodings in command lines cause Bear to fail with an unhandled exception

### ISSUE THREAD

**Issue by cgull** [rizsotto/Bear#141]
I found an issue when trying to use `oclint` on https://github.com/mobile-shell/mosh.  One of Mosh's tests checks for bad UTF-8 handling, and runs a command with an illegal UTF-8 byte sequence in its arguments.  See https://github.com/mobile-shell/mosh/blob/master/src/tests/prediction-unicode.test#L74 and the attached base64-encoded `.cmd` file to see what's going on.  Bear throws an exception when it tries json.dump() on this.  

``` python
bear: Something unexpected had happened.
Traceback (most recent call last):
  File "/home/cgull/src/Bear/test/utf8/../../bear/bear", line 65, in main
    return capture(args)
  File "/home/cgull/src/Bear/test/utf8/../../bear/bear", line 118, in capture
    json.dump(list(entries), handle, sort_keys=True, indent=4)
  File "/usr/lib/python2.7/json/__init__.py", line 189, in dump
    for chunk in iterable:
  File "/usr/lib/python2.7/json/encoder.py", line 431, in _iterencode
    for chunk in _iterencode_list(o, _current_indent_level):
  File "/usr/lib/python2.7/json/encoder.py", line 332, in _iterencode_list
    for chunk in chunks:
  File "/usr/lib/python2.7/json/encoder.py", line 390, in _iterencode_dict
    yield _encoder(value)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xfc in position 12: invalid start byte
```

[10438.cmd.txt](https://github.com/rizsotto/Bear/files/505244/10438.cmd.txt)

I wrote a somewhat-different test case (PR #140) and then realized that fixing this is out of scope for me:  deciding what to do with illegal UTF-8 sequences and serializing them usefully in JSON to oclint or other tooling is an architectural and interop issue.

I originally encountered this on 2.1.5 but it's in github master too, obviously.  This is not a showstopper for Mosh at all, we don't need to run that test at all for `oclint`.

**Comment by rizsotto** [comment]
thank you for the report.

**Comment by rizsotto** [comment]
Fixed in 2.3.3. Sorry for the delay.
